packages/react-component-library/src/components/Field/partials/StyledHintText.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+import { selectors } from '@defencedigital/design-tokens'
+
+const { color } = selectors
+
+export const StyledHintText = styled.span`
+  display: inline-block;
+  font-size: 12px;
```

Comment:
Should we use a typography token for this (`s` I think)?

(I admit I was never convinced by them, but we do have them...)